x/evmutil/keeper/bank_keeper.go

```diff
@@ -0,0 +1,238 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	evmtypes "github.com/tharsis/ethermint/x/evm/types"
+
+	"github.com/kava-labs/kava/x/evmutil/types"
+)
+
+const (
+	// EvmDenom is the gas denom used by the evm
+	EvmDenom = "akava"
+
+	// CosmosDenom is the gas denom used by the kava app
+	CosmosDenom = "ukava"
+)
+
+// ConversionMultiplier is the conversion multiplier between akava and ukava
+var ConversionMultiplier = sdk.NewInt(1_000_000_000_000)
+
+var _ evmtypes.BankKeeper = EvmBankKeeper{}
+
+// EvmBankKeeper is a BankKeeper wrapper for the x/evm module to allow the use
+// of the 18 decimal akava coin on the evm.
+// x/evm consumes gas and send coins by minting and burning akava coins in its module
+// account and then sending the funds to the target account.
+// This keeper uses both the ukava coin and a separate akava balance to manage the
+// extra percision needed by the evm.
+type EvmBankKeeper struct {
+	akavaKeeper Keeper
+	bk          types.BankKeeper
+	ak          types.AccountKeeper
+}
+
+func NewEvmBankKeeper(akavaKeeper Keeper, bk types.BankKeeper, ak types.AccountKeeper) EvmBankKeeper {
+	return EvmBankKeeper{
+		akavaKeeper: akavaKeeper,
+		bk:          bk,
+		ak:          ak,
+	}
+}
+
+// GetBalance returns the total **spendable** balance of akava for a given account by address.
+func (k EvmBankKeeper) GetBalance(ctx sdk.Context, addr sdk.AccAddress, denom string) sdk.Coin {
```

Comment:
Overall implementation looks good- tested in kvtool and balances looked correct - also fixes the gas issue 🎉. 

A couple non-blocking notes - 

* I think `ConvertAkavaToUkava` and `ConvertOneUkavaToAkava` would benefit from table driven tests
* Would like to see unit tests for `SendBalance` and `SetBalance`
* We should add a spec - would especially like a description or diagram that summarizes transaction flow through the module account